src/components/Overview.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div id="overview">
+    <div class="row">
+      <div class="col-xs-4 col-md-12">
+        <h3>Last Meal</h3>
+        <span>{{lastMealName}}</span>
+      </div>
+
+      <div class="col-xs-4 col-md-12">
+        <h3>Most Prepared</h3>
+        <ul class="list-unstyled" v-for="(meal, i) in mostPrepared" :key="i">
+          <li>{{meal.name}}</li>
+        </ul>
+      </div>
+
+      <div class="col-xs-4 col-md-12">
+        <h3>Least Prepared</h3>
+        <ul class="list-unstyled" v-for="(meal, i) in leastPrepared" :key="i">
+          <li>{{meal.name}}</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import {last, map, pipe, reverse, slice} from 'ramda';
+  import {sortMostUsed, sortRecentlyPrepared} from"@/business/sorting";
+
+  const getFirstThree = slice(0, 3);
+
+  export default {
+    name: 'Overview',
+    props: ["meals"],
+    computed: {
+    	mostPrepared: function() {
+        return pipe(sortMostUsed, getFirstThree)(this.meals);
+      },
+      leastPrepared: function() {
+        return pipe(sortMostUsed, reverse, getFirstThree)(this.meals)
+      },
+      lastMealName: function() {
+    		const lastMeal = sortRecentlyPrepared(this.meals)[0];
+
+    		if(lastMeal && lastMeal.hasOwnProperty("name")) {
+    			return lastMeal.name;
+        } else {
+    			return "";
+        }
+      }
+    }
+  }
+</script>
+
+<style scoped lang="less">
```

Comment:
Just curious - when you specify it this way, in Vue, does the `less` get transpiled to `css` at run-time? or at compile time?

Reply:
compile time

Reply:
That - is super cool.

Reply:
inorite!?